specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json

```diff
@@ -1473,12 +1572,12 @@
         }
       },
       "put": {
-        "operationId": "Devices_UpdateCloudProperties",
+        "operationId": "Devices_SetCloudProperties",
```

Comment:
Usually `PUT` operations use a prefix of `Create`, so `Devices_CreateCloudProperties`.  Comment applies to all `PUT` operations.

Reply:
We will address this in the future. This is out of scope for this PR

Reply:
Fixing it later is disruptive and introduces a breaking change for no good reason.  Please correctly name the operation IDs before releasing.

Reply:
What is the status on fixing the operation IDs?

Reply:
hi @jhendrixMSFT, these PUT operations stands for CreateOrUpdate, and we have a Patch which have operation id Devices_Updatexxx, to avoid conflict, Set is more proper meaning for us, put a new one or replace the existed one, what do you suggest?  Checked through cosmos db, digital twin and hub arm API, some was use add, some was use createorupdate, some us created, do we have a documentation to follow on this?

Reply:
I've updated most of the operation IDs. The only ones that still say Set are properties and cloud properties. These are concepts in IoT Central that come with new devices. There isn't a concept of "creating properties" since they already exist. They are set with a new value, so keeping the terminology as "set" for those.

Reply:
Makes sense, thanks for the explanation.